include/aikido/robot/util.hpp

```diff
@@ -24,14 +24,13 @@ namespace util {
 
 struct VectorFieldPlannerParameters
 {
-  VectorFieldPlannerParameters() = default;
   VectorFieldPlannerParameters(
-      double linearVelocity,
-      double negativeDistanceTolerance,
-      double positiveDistanceTolerance,
-      double initialStepSize,
-      double jointLimitTolerance,
-      double constraintCheckResolution,
+      double linearVelocity = 0.2,
```

Comment:
This is part of https://github.com/personalrobotics/aikido/pull/382, so let's remove this file from this PR.